embuilder.py

```diff
@@ -277,19 +279,19 @@ def main():
     elif what == 'cocos2d':
       build_port('cocos2d', None, ['-s', 'USE_COCOS2D=3', '-s', 'USE_ZLIB=1', '-s', 'USE_LIBPNG=1'])
     else:
-      shared.logging.error('unfamiliar build target: ' + what)
```

Comment:
This file still has references to shared.logging, should we want to update those too?

Reply:
Yes. Done.